sclack/components.py

```diff
@@ -200,7 +200,7 @@ def keypress(self, size, key):
         if key == keymap['go_to_sidebar']:
             self.header.restore_topic()
             urwid.emit_signal(self, 'go_to_sidebar')
-            return True
+            return key
```

Comment:
Is this really necessary?

Reply:
this line `urwid.emit_signal(self, 'go_to_sidebar')` return the `key` so `return True` neither `return key` is really necessary...

Reply:
We should keep `return True` for 2 main reasons:
- Keeping them standard
- Avoiding the propagation of the event to the top-components (when they exist). That's a common source of bugs where some events are handled by two components.

Reply:
It happened on #5 too.

Reply:
Gotcha! I'll find another solution...

Reply:
I remove this if.